<<<<<<< HEAD

max_acceleration_xy: 0.5  # m/s^2
max_acceleration_theta: 6.0  # rad/s^2

max_velocity_xy: 0.4  # m/s^2
max_velocity_theta: 6.0  # rad/s^2

control_range_xy: 1.4
control_a_xy: 1.4
control_a_theta: 0.5
=======
hard_limit_acceleration_xy: 3.0 # m/s^2
soft_limit_acceleration_xy: 2.5 # m/s^2
hard_limit_acceleration_theta: 5.0 # rad/s^2
soft_limit_acceleration_theta: 4.0 # rad/s^2
hard_limit_velocity_xy: 3.0 # m/s^2
soft_limit_velocity_xy: 2.5 # m/s^2
hard_limit_velocity_theta: 5.0 # rad/s^2
soft_limit_velocity_theta: 4.0 # rad/s^2

control_range_xy: 1.0
control_a_xy: 1.0
control_a_theta: 1.0
>>>>>>> origin/main

p_gain_xy: 1.5
d_gain_xy: 0.0
p_gain_theta: 2.5
d_gain_theta: 0.0
delayfactor_sec: 0.0
